code/ARAX/ARAXQuery/Overlay/Query_ICEES.py

```diff
@@ -0,0 +1,200 @@
+'''This module defines the class Query_ICEES to
+communicate with the ICEES API(3.0.0) to obtain drug - chemical-substance association
+between PUBCHEM:2083 and PUBCHEM:281.
+
+'''
+
+__author__ = 'Mayank Murali'
+__copyright__ = 'The Pennsylvania State University'
+__credits__ = ['Mayank Murali', 'David Koslicki']
+__license__ = 'MIT'
+__version__ = '0.1.0'
+__maintainer__ = ''
+__email__ = ''
+__status__ = 'Trial'
+
+import requests
+import re
+import json
+import sys
+import urllib.parse
+import unittest
+
+#from cache_control_helper import CacheControlHelper
+
+# input json file
+with open('Input.json') as f1:
+    input_data = json.load(f1)
+#print(input_data)
+
+# response json file
+with open('response_1593100927743.json') as f2:
+    output_data = json.load(f2)
+   
```

Comment:
These lines (25-32) are causing errors when I try to use `Query_ICEES.post_knowledge_graph_overlay()`:
```
  File "/Users/aglen/translator/RTX_new/RTX/code/ARAX/ARAXQuery/Overlay/Query_ICEES.py", line 26, in <module>
    with open('Input.json') as f1:
FileNotFoundError: [Errno 2] No such file or directory: 'Input.json'
```
```
  File "/Users/aglen/translator/RTX_new/RTX/code/ARAX/ARAXQuery/Overlay/Query_ICEES.py", line 31, in <module>
    with open('response_1593100927743.json') as f2:
FileNotFoundError: [Errno 2] No such file or directory: 'response_1593100927743.json'
```
If I comment them out, `Query_ICEES.post_knowledge_graph_overlay()` runs successfully. I see it looks like you use these missing files for testing - perhaps you could modify your test so that it doesn't rely on additional files?

Reply:
Thank you for looking into the code @amykglen. 'Input.json' and 'response..json' were provided as an addition to run a particular query. I can extract it and add it to the code.

Reply:
And note @MayankMurali, as specified when I assigned this task, these are _examples_ of input and output. The actual input and output will be processed [here](https://github.com/RTXteam/RTX/blob/issue851/code/ARAX/ARAXQuery/Overlay/overlay_exposures_data.py#L67-L95), so there should be no hard-coded JSON in your `Query_ICEES` (i.e. it should expect and return JSON, not read it from a file). Just to clarify...

Reply:
@amykglen, I have removed the JSON file input part. The code should be able to run the query when passed to 'Query_ICEES.post_knowledge_graph_overlay' as a parameter.

Also, @dkoslicki I'll add in the code for other methods (get_feature_identifiers,...) in case they are required.